### DISCUSSION TITLE
Formula fields

### DISCUSSION THREAD

**Discussion by hmirzoyan-chwy** [directus/directus#6120] [Feature Requests]
It would be great to be able to create a field that is formula-based. 

So I can specify a field to also include another field as variables and maybe do some basic math or array functions.

**Comment by wendevlin** [comment]
Hey @hmirzoyan-chwy,

Directus is very modular, so if there is no field you need you can create your own.
Checkout the documentation for interfaces: https://docs.directus.io/concepts/interfaces/

Here you see how to create your own interface: https://docs.directus.io/guides/interfaces/

I hope this helps :slightly_smiling_face:

**Comment by benhaynes** [comment]
Love it! But duplicate of: https://github.com/directus/directus/discussions/3079
